### PR TITLE
ISSUE-381 | Replace dummy text in `APP EXPLANATION` on Apple Watch

### PR DESCRIPTION
**Pull request by bashjs**
#381

## Blocked by

- [ ] Add [localization support](https://developer.apple.com/forums/thread/124254)

## Changes

- Replaced `Bla bla {Record,Share,Update}` text with `For the iAPS app to work on the Apple Watch, please ensure that you have enabled HealthKit permissions.`

### CODE REVIEW

File: FreeAPSWatch WatchKit Extension/Info.plist [hunk 1]
```diff
@@ -3,11 +3,11 @@
 <plist version="1.0">
```
Comment (bashjs) [inline FreeAPSWatch WatchKit Extension/Info.plist:3]: What if we delete this file? Will the app pick [these translations automatically?](https://github.com/Artificial-Pancreas/iAPS/blob/dev/FreeAPS/Resources/Info.plist#L77-L80)  🤔

Comment (Jon-b-m) [reply FreeAPSWatch WatchKit Extension/Info.plist:3]: I'm not sure, but I remember when it was needed to add this file, hence I would suggest not to delete it. Speaking of translations, you didn't add the English source strings for the localization.

**Comment by bjornoleh** [comment]
Hi, the suggested changes are not quite how these strings are meant to be. Please see some examples here:

https://github.com/Artificial-Pancreas/iAPS/blob/cb67cbef100253c965f0a30cc2582cb1260201b0/FreeAPS/Resources/Info.plist#L77

These strings are presented tu the user when the app requests permissions. The idea is that the app is transparent about why it wants you to give permissions, such as collect sensitive health data or usage of capabilities like bluetooth. For an app to pass AppStore review (not relevant for us), these things can typically be a blocker unless its done properly.

Documentation is available here:
https://developer.apple.com/documentation/bundleresources/information_property_list/nshealthupdateusagedescription

**Comment by Jon-b-m** [comment]
It looks ok, but you just need to create the localizable strings also. If you create the English source file with strings  first you after that can just check all languages for the other to be created in Xcode. I can help you with this next week after Christmas.

**Comment by Jon-b-m** [comment]
Please check out this PR: https://github.com/Artificial-Pancreas/iAPS/pull/670
and test it this fixes it for you?

**Comment by Jon-b-m** [comment]
Looks like there are still some strings missing in the localizable.strings ?

**Pull request closed without merging**